scrapy/utils/spider.py

```diff
@@ -4,15 +4,11 @@
 import six
 
 from scrapy.spiders import Spider
-from scrapy.utils.misc import  arg_to_iter
+from scrapy.utils.misc import arg_to_iter as iterate_spider_output
```

Comment:
if we go this way it worths adding a comment about why is there an unused import